app/usr/lib/systemd/system/pulp_resource_manager.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
```

Comment:
@dralley 

`git mv` was used, but from some reason for these two files it after some point started showing as delete + create probably the files changed above some threshold? 

```
[fdobrovo@gilgamesh pulp]$ git commit -a --amend
[systemd d000d39] Update systemd tasking service files
 Date: Wed Apr 12 14:59:43 2017 +0200
 8 files changed, 36 insertions(+), 30 deletions(-)
 rename server/etc/default/systemd_pulp_celerybeat => app/etc/default/pulp_celerybeat (100%)
 rename server/etc/default/systemd_pulp_resource_manager => app/etc/default/pulp_resource_manager (100%)
 rename server/etc/default/systemd_pulp_workers => app/etc/default/pulp_workers (100%)
 rename {server => app}/usr/lib/systemd/system/pulp_celerybeat.service (69%)
 create mode 100644 app/usr/lib/systemd/system/pulp_resource_manager.service
 create mode 100644 app/usr/lib/systemd/system/pulp_worker@.service
 delete mode 100644 server/usr/lib/systemd/system/pulp_resource_manager.service
 delete mode 100644 server/usr/lib/systemd/system/pulp_workers.service
```